sentence.go

```diff
@@ -343,3 +432,91 @@ func Parse(raw string) (Sentence, error) {
 	}
 	return nil, &NotSupportedError{Prefix: s.Prefix()}
 }
+
+// SupportedParsers list all available parsers provided by the library
+var SupportedParsers = map[string]ParserFunc{
```

Comment:
I don't think we should export this.

Reply:
I'll remove that. This is my selfish need to have up to date list of supporter sentences.